challenge_6.js

```diff
@@ -6,11 +6,28 @@
  */
 exports.SumOfASquare = SumOfASquare;
 
-function SumOfASquare(naturalNumbers){
+function SumOfASquare(naturalNumbers) {
   this.naturalNumbers = naturalNumbers;
 }
 
 // do work here
+SumOfASquare.prototype.sumOfSquares = function () {
+  let sum = 0;
+  for (let i = 1; i <= this.naturalNumbers; i++) {
+    let squared = Math.pow(i, 2);
```

Comment:
You can skip the step of saving Math.pow(i,2) into `squared` variable and set it to the `sum` variable below.